governance/default.go

```diff
@@ -518,33 +518,64 @@ func (g *Governance) ClearVotes(num uint64) {
 	logger.Info("Governance votes are cleared", "num", num)
 }
 
-// parseVoteValue parse vote.Value from []uint8 to appropriate type
+// parseVoteValue parse vote.Value from []uint8, [][]uint8 to appropriate type
 func (g *Governance) ParseVoteValue(gVote *GovernanceVote) (*GovernanceVote, error) {
 	var val interface{}
-	k := GovernanceKeyMap[gVote.Key]
-
-	// filter out if vote value is an interface list
-	if reflect.TypeOf(gVote.Value) == reflect.TypeOf([]interface{}{}) {
-		return nil, ErrValueTypeMismatch
+	k, ok := GovernanceKeyMap[gVote.Key]
+	if !ok {
+		logger.Warn("Unknown key was given", "key", k)
+		return nil, ErrUnknownKey
 	}
 
 	switch k {
 	case params.GovernanceMode, params.MintingAmount, params.MinimumStake, params.Ratio:
-		val = string(gVote.Value.([]uint8))
+		v, ok := gVote.Value.([]uint8)
+		if !ok {
+			return nil, ErrValueTypeMismatch
+		}
+		val = string(v)
 	case params.GoverningNode, params.AddValidator, params.RemoveValidator:
-		val = common.BytesToAddress(gVote.Value.([]uint8))
+		if v, ok := gVote.Value.([]uint8); ok {
```

Comment:
Does `params.GoverningNode` also allows arrays?

Reply:
@aidan-kwon `params.GoverningNode` does not allow arrays. Thanks for the advice. It is reflected at https://github.com/klaytn/klaytn/pull/1110/commits/81771918b0d94ab90c7f28fc8df92912dbe6280d